csv_detective/detect_fields/other/float/__init__.py

```diff
@@ -2,11 +2,12 @@
 
 
 def _is(val):
-    '''Detects floats'''
+    '''Detects floats, assuming that tables will not have scientific
+    notations (3e6) or "+" in the string. "-" is still accepted.'''
     try:
-        if '_' in val:
+        if any([k in val for k in ['_', '+', 'e', 'E']]):
             return False
-        float(val.replace(' ', '').replace(',', '.'))
+        float(val.replace(',', '.'))
```

Comment:
Since we need to apply the same transformation between `_is` and the `create_profile` function, could we turn this into a util function to be used in both cases?
Since we made update the casting logic in the future :)